src/components/policy/policy_regular/src/policy_table/validation.cc

```diff
@@ -115,8 +115,13 @@ bool ApplicationPoliciesSection::Validate() const {
 }
 
 bool ApplicationParams::Validate() const {
+  // Check for empty "groups" sub-sections
+  if (groups.empty()) {
```

Comment:
@LuxoftAKutsan yes, it could be checked with template paramter, however now `groups` have a type  `Array<String<1, 255>, 0, 255> Strings` so if I change this type to `Array<String<1, 255>, 1, 255> Groups;` to check empty alowness, it will lead to a lot of changes in a lot of functions where it used. That's why it's safer just to add this check to `ApplicationParams::Validate()`